srv/salt/_modules/rgw.py

```diff
@@ -125,6 +125,16 @@ def create_bucket(**kwargs):
         return False
     return True
 
+def lookup_bucket(user, bucket):
+    s3conn = boto.connect_s3(
+        aws_access_key_id=access_key(user),
+        aws_access_key_id=secret_key(user),
+    )
+
```

Comment:
we could make S3Conn as a class, and use the above create and lookup as methods?